fabtests/common/shared.c

```diff
@@ -2880,7 +2880,8 @@ void ft_hmem_usage(void)
 	FT_PRINT_OPTS_USAGE("-D <device_iface>", "Specify device interface: eg ze, cuda, or rocr (default: None). "
 			     "Automatically enables FI_HMEM (-H)");
 	FT_PRINT_OPTS_USAGE("-i <device_id>", "Specify which device to use (default: 0)");
-	FT_PRINT_OPTS_USAGE("-H", "Enable provider FI_HMEM support");
+	FT_PRINT_OPTS_USAGE("-H", "Enable provider FI_HMEM support.");
+	FT_PRINT_OPTS_USAGE("", "For some tests, source address must be provided if FI_HMEM is enabled.");
```

Comment:
Do the tests that require OOB setup fail or otherwise provide feedback to the user?  How does a user know what "some tests" refer to?

Reply:
Will look into fixing.